docs/optimization_guide/nncf/ptq/code/ptq_openvino.py

```diff
@@ -15,7 +15,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # openvino.runtime.Model object
+import openvino as ov
+model = ov.Model() # openvino.runtime.Model object
```

Comment:
```suggestion
import openvino.runtime as ov
model = ov.Core().read_model("model path")
```

remove `import openvino.runtime as ov` from the 25 line